tools/webpack-helpers.ts

```diff
@@ -0,0 +1,72 @@
+import { basename } from 'path';
+
+/* Return imports-loader format to the config
+    For example:
+    [
+        // Adds 'imports-loader?this=>widndow'
+        {path: './foler/my_module.js', args: '?this=>window'},
+    ]
+*/
+interface importLoaderOptions {
+   path: string;
+   args: string;
+}
+function getImportLoaders (optionsArr:Array<importLoaderOptions>) {
```

Comment:
Mostly nit-picking, there is a extra space after function name, and it think it looks clean is so have space before the type annotation.